### ISSUE TITLE
date de dernière modification d'une fiche

### ISSUE THREAD

**Issue by py-b** [InseeFrLab/utilitR#357]
À ce stade, la seule façon est de regarder si les fichiers sources ont été modifiés le dépôt. C'est un peu technique pour qui n'est pas familier.

On pourrait envisager d'ajouter une petite note _"Dernière modification de cette fiche le ..."_ sur le site web. Certainement un jeu d'enfant pour @linogaliana ?

**Comment by sylvainDaubree** [comment]
Est-ce que le plus pertinent ne serait pas de mettre un lien "historique de la fiche" qui pointerait sur l'historique github plutôt ? J'ai peur que ce ne soit pas super pertinent la date brute de dernière modif (on a pu passer pour rajouter une virgule et laisser un bon gros deprecated à côté, si les gens prennent ça pour "l'info date d'avant-hier, c'est du solide", ce serait dommage). Ou alors, on prend le problème autrement en se disant que l'engagement qu'on prend dans le projet c'est de relire l'ensemble du bazar tous les ans, sans forcément faire de modif, mais juste vérifier que ça correspond toujours à l'état de l'art. Et du coup on peut mettre une date un peu partout "validé le XXXXXX". 

Bon c'est encore un peu brouillon comme idée mais j'ai l'impression qu'on ne peut pas évacuer le problème de "comment savoir que c'est à jour" juste avec un script automatique.

**Comment by py-b** [comment]
On peut faire les 2 : une date + un lien vers les commits sur la fiche pour qui désire en savoir davantage.

**Comment by py-b** [comment]
Ce ne serait pas ce que fait cette ligne : https://github.com/InseeFrLab/utilitR/blob/e7b5e615243dc08e73e345a5a9335f83b1068b70/_bookdown.yml#L46 ?

Si oui, pourquoi le lien vers l'historique des commits du fichier Rmd n'apparaissent pas sur le site ? :thinking:

**Comment by py-b** [comment]
Pour mémoire, les 3 lignes suivantes sont sans effet depuis l'utilisation de bs4 (merci @RLesur) :
https://github.com/InseeFrLab/utilitR/blob/1acfefb885506db3774ecf3a428edc447ba3ee10/_bookdown.yml#L44-L46

`view` et `edit` sont désormais générés via 
https://github.com/InseeFrLab/utilitR/blob/1acfefb885506db3774ecf3a428edc447ba3ee10/_output.yml#L19-L20

À voir s'il faut creuser dans cette direction.

**Comment by ddotta** [comment]
Oui, dans le template original de bs4_book, il n'est prévu que ces 2 éléments

https://github.com/rstudio/bookdown/blob/4ac16f08e75c39b860562328632f95d2742452c6/inst/templates/bs4_book.html#L127-L134

**Comment by linogaliana** [comment]
Discussion à suivre [ici](https://github.com/quarto-dev/quarto-cli/issues/5967). Idée: utiliser la date du dernier commit

**Comment by linogaliana** [comment]
[Solution à tester depuis GHA](https://github.com/adityam/stochastic-control/blob/quarto/.github/workflows/quarto-render.yml#L24)

**Comment by linogaliana** [comment]
@py-b j'ai fait [ça](https://pythonds.linogaliana.fr/content/manipulation/02a_pandas_tutorial.html#informations-additionnelles) pour mon cours de python, ça conviendrait ?

**Comment by py-b** [comment]
oui, très bien !
